woocommerce/woocommerce-android
# Feature/store creation country picker

Closes: #8166


### Description


**Disclaimer**: Please don't panic about the big number of file changes. Most of the changes are small simple one due to minor refactors in navigation, or renaming stuff. The big changes are in the new files added for the country picker step.

Adds a country selector step to store creation following this design:

![Screenshot 2023-01-11 at 21 26 38](https://user-images.githubusercontent.com/2663464/211910645-52918b4a-a157-4a62-b12d-9080b33b00db.png)

- The list of countries is hardcoded into the app for now as we do not have any available API request to fetch a list of available countries when there is no site selected. 

### Testing instructions
- Compile the app using `vanillaDebug` flavour
- Trigger store creation and navigate to country picker step
- Select a country
- Verify that the selected country is displayed in Store summary step, which is displayed right after domain picker step: 


https://user-images.githubusercontent.com/2663464/211911380-ff5dd188-c2fa-4f94-a04d-fa38a861d3a4.mp4



### Images/gif
